WordPress/Classes/ViewRelated/Aztec/Stock Photos/AztecMoreCoordinator.swift

```diff
@@ -0,0 +1,48 @@
+import MobileCoreServices
+
+/// Prepares the alert controller that will be presented when tapping the "more" button in Aztec's Format Bar
+final class AztecMoreCoordinator {
+    func present(origin: UIViewController & UIDocumentPickerDelegate, view: UIView) {
+        let alertController = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+
+        alertController.addAction(freePhoto())
+        alertController.addAction(files(origin: origin))
+        alertController.addAction(cancel())
+
+
+        alertController.popoverPresentationController?.sourceView = view
+        alertController.popoverPresentationController?.sourceRect = CGRect(origin: view.frame.origin, size: CGSize(width: 1, height: 1))
+        alertController.popoverPresentationController?.permittedArrowDirections = .any
+
+        origin.present(alertController, animated: true, completion: nil)
+    }
+
+    private func freePhoto() -> UIAlertAction {
```

Comment:
Maybe this func could be called `freePhotoAction` or `freePhotoAlertAction`? 
At first glance it was a bit confusing, but the return value makes it clear so it's up to you.

Reply:
I "optimized" for readability at the point of usage:
``
alertController.addAction(freePhoto())
``

That, to me, read better than 

``
alertController.addAction(freePhotoAction())
``

But I see your point. Will rename!